Dockerfile.ocp

```diff
@@ -1,14 +1,15 @@
 FROM openshift/origin-base AS builder
 
-RUN yum install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools
-
-RUN dd bs=1024 count=3200 if=/dev/zero of=esp.img && \
+RUN if [ "$ARCH" == "x86_64" ]; then \
+      yum install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools && \
+      dd bs=1024 count=3200 if=/dev/zero of=esp.img && \
       mkfs.msdos -F 12 -n 'ESP_IMAGE' ./esp.img && \
       mmd -i esp.img EFI && \
       mmd -i esp.img EFI/BOOT && \
       mcopy -i esp.img -v /boot/efi/EFI/BOOT/BOOTX64.EFI ::EFI/BOOT && \
       mcopy -i esp.img -v /boot/efi/EFI/redhat/grubx64.efi ::EFI/BOOT && \
-      mdir -i esp.img ::EFI/BOOT
+      mdir -i esp.img ::EFI/BOOT; \
```

Comment:
you shouldn't need  a semi-colon there....

Reply:
whoops! left it when splitting lines

Reply:
actually this is a continuation of the precedent line, so the semi-colon should be there

Reply:
ahh, I see it now. Interesting that they are &&\'ed. but *shrug*